src/puppetlabs/jdbc_util/core.clj

```diff
@@ -139,7 +140,9 @@
       (let [tail (.substring s (.length ^String match) (.length s))
             replaced (str/replace match #"(.*)\?$" (str "$1" replacement))]
         (str replaced tail))
-      (throw (IllegalArgumentException. (str "there are not " n " '?'s in the given string"))))))
+      (throw (IllegalArgumentException. (trun "There is not one '?' in the given string"
```

Comment:
Not clear how/where this exception will be handled, but seemed more likely to be something a user might see.  I agree that it isn't terribly important in the grand scheme of things.  I can change it to be more simple.